### DISCUSSION TITLE
Omit the rest of the table rename cases related to the table when the answer is `yes`

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#832] [Ideas]
Lets assume we have Four tables as Follows :
`TableA, TableB, TableC and TableD`
Now, due to some reason , they have to rename the tables as Follows:
`RTableA, RTableB, RTableC and RTableD`

Now, During migrations piccolo asks 16 questions .
But the number of question can be minimized  to any number between 4 -16 , by simply omitting the question when the answer is yes.

Like as Follows:-
`Did you rename TableB (tablename: table_b) to RTableA (tablename: r_table_a)? (y/N)`
The answer is `N`.
Now. Piccolo asks follows:
 `Did you rename TableA (tablename: table_a) to RTableA (tablename: r_table_a)? (y/N)`
The answer is 'y'.
When the answer is 'y' for the table `RTableA`, piccolo should stop asking the following question, because answering the following question is unnecessary and illogical.
 
`Did you rename TableC (tablename: table_c) to RTableA (tablename: r_table_a)? (y/N)`
`Did you rename TableD (tablename: table_d) to RTableA (tablename: r_table_a)? (y/N)`

**Comment by dantownsend** [comment]
@sumitsharansatsangi Interesting - thanks for reporting this. I'll create a demo project and give it a go.

**Reply by dantownsend** [reply]
@sumitsharansatsangi You're right - it is pretty annoying.

In this example below, even though I said that `TableB` had been renamed, it kept asking me:

<img width="737" alt="Screenshot 2023-05-28 at 09 25 24" src="https://github.com/piccolo-orm/piccolo/assets/350976/c0d68322-1009-46e6-b747-ab8e8bfebfcd">

**Reply by dantownsend** [reply]
I think I have a solution to this - will turn it into an issue, then will create a PR.

**Reply by sumitsharansatsangi** [reply]
Yeah, It's true @dantownsend. It is asking again and again for `TableB`. I haven't noticed. 
Please fix it. It will be pretty helpful.

**Reply by dantownsend** [reply]
This is fixed in ``piccolo==0.113.0``.
